lib/decompress/zstd_decompress_block.c

```diff
@@ -563,13 +563,13 @@ typedef struct {
 } ZSTD_fseState;
 
 typedef struct {
-    BIT_DStream_t DStream;
     ZSTD_fseState stateLL;
     ZSTD_fseState stateOffb;
     ZSTD_fseState stateML;
-    size_t prevOffset[ZSTD_REP_NUM];
     const BYTE* prefixStart;
     const BYTE* dictEnd;
+    BIT_DStream_t DStream;
+    size_t prevOffset[ZSTD_REP_NUM];
```

Comment:
I've measure that this helps gcc a bit and hurts clang a bit on x86 with clang-9.0.1 and gcc-9.2.0.